lib/dor/workflow_exception.rb

```diff
@@ -0,0 +1,4 @@
+module Dor
+  class WorkflowException < ::RuntimeError
+  end
+end
```

Comment:
Do downstream applications care? The important thing for me is downstream applications aren't trying to catch `RestClient` or `Faraday` exceptions.